gym/envs/mujoco/humanoidstand.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np
+from gym.envs.mujoco import mujoco_env
+from gym import utils
+
+def mass_center(model):
+    mass = model.body_mass
+    xpos = model.data.xipos
+    return (np.sum(mass * xpos, 0) / np.sum(mass))[0]
+
+class HumanoidStandEnv(mujoco_env.MujocoEnv, utils.EzPickle):
+    def __init__(self):
+        mujoco_env.MujocoEnv.__init__(self, 'humanoid.xml', 5)
+        utils.EzPickle.__init__(self)
+
+    def _get_obs(self):
+        data = self.model.data
+        return np.concatenate([data.qpos.flat[2:],
+                               data.qvel.flat,
+                               data.cinert.flat,
+                               data.cvel.flat,
+                               data.qfrc_actuator.flat,
+                               data.cfrc_ext.flat])
+
+    def _step(self, a):
+        pos_before = mass_center(self.model)
+        self.do_simulation(a, self.frame_skip)
+        pos_after = mass_center(self.model)
+        alive_bonus = 15.0
+        data = self.model.data
+        lin_vel_cost = 0.25 * (pos_after - pos_before) / self.model.opt.timestep
+        quad_ctrl_cost =  0.1 * np.square(data.ctrl).sum()
+        quad_impact_cost = .5e-6 * np.square(data.cfrc_ext).sum()
+        quad_impact_cost = min(quad_impact_cost, 10)
+        reward = alive_bonus - lin_vel_cost - quad_ctrl_cost - quad_impact_cost 
```

Comment:
Subtracing lin_vel_cost will give rewards when it walks backwards, which probably isn't what you want.